gcexport.py

```diff
@@ -652,28 +719,28 @@ def export_data_file(activity_id, activity_details, args, file_time, append_desc
         makedirs(directory)
 
     # timestamp as prefix for filename
-    if args.fileprefix > 0:
-        prefix = "{}-".format(start_time_locale.replace("-", "").replace(":", b"").replace(" ", "-"))
+    if args.fileprefix and args.fileprefix > 0:
+        prefix = "{}-".format(start_time_locale.replace("-", "").replace(":", "").replace(" ", "-"))
```

Comment:
```suggestion
        prefix = f"{start_time_locale.replace("-", "").replace(":", "").replace(" ", "-")}-"
```

Reply:
Python2 compatibility update
```suggestion
        prefix = start_time_locale.replace("-", "").replace(":", "").replace(" ", "-") + "-"
```

Reply:
This one is missing, I think.